scout/constants/case_tags.py

```diff
@@ -13,6 +14,7 @@
         "format": "PDF",
         "pdf_export": False,
     },
+    "reference_info": {"key_name": "reference_info", "format": "YAML", "pdf_export": True},
```

Comment:
Also the PDF export of this one looks messy:

<img width="1311" alt="image" src="https://user-images.githubusercontent.com/28093618/203707501-c39d53a9-5027-429c-8c29-3ff2d36311c7.png">

The easiest solution would be to just set pdf_export to False

Reply:
Mm, I did the same here. It is still not super easy to read, but at least the ws is native:
<img width="1245" alt="Screenshot 2022-11-24 at 08 27 35" src="https://user-images.githubusercontent.com/758570/203720014-57d83597-b1f3-41fe-950b-a8bd023b2e5d.png">
One can still dl the yaml if that is preferred.